index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+    />
+  </head>
+
+  <body>
+    <div id="navbar">
+      <ul>
+        <li><a href="#">About</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#footer-section">Footer</a></li>
+      </ul>
+    </div>
```

Comment:
Please take note of HTML Semantic elements. The div tag should be changed to nav tag. You can read more about them using this link https://www.w3schools.com/html/html5_semantic_elements.asp It makes the page easier to read with greater accessibility.

Reply:
### Great comments ✨ however, it only lacks a little bit of explanation and it can be improved 🚀 

- [ ] As reviewers we also focus on things the student did well on his code so it will be great if you highlight some of the work that the student did well 💪🏼 👏🏼 
- [ ] Kindly describe why the student should use the `nav` tag I recommend reading [why we should use semantics](https://www.thisdot.co/blog/semantic-html-why-it-matters-and-top-tips-on-how-to-apply-it) for more convincing reasons 👍🏼 💯 
- [ ] Examples are stronger than words in conveying messages🎯, so you can add an example To ensure that the student understands what changes he needs to make like the following ⬇️ 
```
<nav id="navbar">
  <ul>
    <li><a href="#">About</a></li>
    <li><a href="#">About</a></li>
    ...
  </ul>
</nav>
```
- [ ] This is **[Optional]** but highly recommended in order for your comments to sound more human-friendly please add some emojis to your comments 🤖 👽

Reply:
## Changes Required ♻♻♻♻♻
Great work so far 👍👍👍👍👍 with your html file. Tags are nicely arranged but there are small changes required from you 👌. Use nav tag instead of a div tag for the navigation. This will make your code easier to understand, making the source code more readable for other developers 🔨 🔨 🔨 🔨 🔨 🔨 🔨 
```suggestion
    <nav id="navbar">
      <ul>
        <li><a href="#">About</a></li>
        <li><a href="#">About</a></li>
        <li><a href="#">About</a></li>
        <li><a href="#footer-section">Footer</a></li>
      </ul>
    </nav>
```

Reply:
[OPTIONAL] I kindly suggest that it's better to use a polite suggestive tone in reviews because sometimes there are many ways to implement the requirement so saying that they must "use" might limit their expertise, growth, exploration, or creativity 😉